lib/Cake/Core/App.php

```diff
@@ -889,6 +889,31 @@ protected static function _packageFormat() {
 	}
 
 /**
+ * Increases the PHP "memory_limit" ini setting by the specified amount
+ * in megabytes  
+ *  
+ * @param string $additionalMb Number in megabytes
+ * @return void
+ */
+	public static function increasePhpMemoryLimit($additionalMb) {
```

Comment:
"Php" can be dropped from the method name and shortened to `increaseMemoryLimit`. Also imo the argument should not force specifying additional memory in Mb only.